Characters/Player/Player.cs

```diff
@@ -10,10 +10,20 @@ public class Player : KinematicBody2D
 	private int gravity = 1000;
 	private float friction = 0.08f;
 	private bool jumping = false;
+	private Label speedValue;
+	private Label jumpSpeedValue;
+	private Label maxSpeedValue;
+	private Label gravityValue;
+	private Label frictionValue;
 
 	// Called when the node enters the scene tree for the first time.
 	public override void _Ready()
 	{
+		speedValue = GetNode<Label>("/root/MainScene/DeveloperControl/DevInfoContainer/PlayerValues/SpeedControls/SpeedValue");
+		jumpSpeedValue = GetNode<Label>("/root/MainScene/DeveloperControl/DevInfoContainer/PlayerValues/JumpSpeedControls/JumpSpeedValue");
+		maxSpeedValue = GetNode<Label>("/root/MainScene/DeveloperControl/DevInfoContainer/PlayerValues/MaxSpeedControls/MaxSpeedValue");
+		gravityValue = GetNode<Label>("/root/MainScene/DeveloperControl/DevInfoContainer/PlayerValues/GravityControls/GravityValue");
+		frictionValue = GetNode<Label>("/root/MainScene/DeveloperControl/DevInfoContainer/PlayerValues/FrictionControls/FrictionValue");
```

Comment:
There must be a better way to reference a node outside of its scene. Some suggest using GetScene GetRoot etc. but some of them are deprecated. Will take a look further later.